doc/api/n-api.md

```diff
@@ -11,6 +11,13 @@ changes in the underlying JavaScript engine and allow modules
 compiled for one version to run on later versions of Node.js without
 recompilation.
 
+As the feature is currently experimental it must be enabled with the
+following command line option:
```

Comment:
Should this link to https://nodejs.org/dist/latest-v8.x/docs/api/cli.html#cli_napi_modules